### PR TITLE
revise web

### PR DESCRIPTION
**Pull request by skzhang1**
## Why are these changes needed?

[1] change website style
[2] record past/future talks

## Checks

- [1] I've included any doc changes needed for https://autogen-ai.github.io/autogen/. See https://autogen-ai.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [2] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [3] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by qingyun-wu** [comment]
Could you fix code formatting issues? Thanks!

**Comment by skzhang1** [comment]
> Could you fix code formatting issues? Thanks!

Done!

**Comment by qingyun-wu** [comment]
We already moved to a new org `autogenhub`. Can you change the places where you use `autogen-ai` to `autogenhub`?

**Comment by qingyun-wu** [comment]
Also, where did you get the two doc files added under groupchat? Did you create the content yourself? 

@skzhang1

File: website/docs/topics/groupchat/transform_messages_speaker_selection.mdx [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
```
Comment (marklysze) [inline website/docs/topics/groupchat/transform_messages_speaker_selection.mdx]: I think this and the resuming_groupchat can be removed from the commit, they are generated as needed.

**Comment by marklysze** [comment]
> Also, where did you get the two doc files added under groupchat? Did you create the content yourself?
> 
> @skzhang1

They can be removed from the commit, they are auto-generated.

**Comment by skzhang1** [comment]
> > Also, where did you get the two doc files added under groupchat? Did you create the content yourself?
> > @skzhang1
> 
> They can be removed from the commit, they are auto-generated.

Yes... I previously thought it was something needed to keep. Removed.

**Comment by skzhang1** [comment]
Hi @marklysze,  can you approve this PR, thanks! I would like to see how the website going with the new version.  We can make further revisions in following PRs.

**Review by qingyun-wu — APPROVED** [review]
Approving now but may need follow-up fixes. Thanks.

**Pull request merged**